litestar/plugins.py

```diff
@@ -90,13 +86,13 @@ def create_dto_for_type(self, field_definition: FieldDefinition) -> type[DTOInte
 
 
 @runtime_checkable
-class OpenAPISchemaPluginProtocol(Protocol[ModelT]):
+class OpenAPISchemaPluginProtocol(Protocol):
     """Plugin to extend the support of OpenAPI schema generation for non-library types."""
 
     __slots__ = ()
 
     @staticmethod
-    def is_plugin_supported_type(value: Any) -> TypeGuard[ModelT]:
+    def is_plugin_supported_type(value: Any) -> bool:
```

Comment:
minor breaking change - see the point regarding the other method. Namely, we are not dealing with a model here at all but rather with a lot of different types.